tests/unit_tests/bittensor_tests/test_neuron.py

```diff
@@ -0,0 +1,60 @@
+import bittensor
+import torch 
+import torch.nn as nn
+from torch.nn import TransformerEncoder, TransformerEncoderLayer
+
+adv_server = bittensor._neuron.text.advanced_server.server()
```

Comment:
Is there a reason this is here, or can you instantiate this within the test function?

Reply:
yup, it should be in the test function.